### PR TITLE
feat: Use ens API

### PR DESCRIPTION
**Pull request by alaibe**
New ENS API

https://github.com/status-im/status-go/pull/2471
https://github.com/status-im/status-lib/pull/143

### CODE REVIEW

**Review by richard-ramos — APPROVED** [review]

File: ui/app/AppLayouts/Profile/stores/RootStore.qml [hunk 1]
```diff
@@ -247,7 +247,7 @@ QtObject {
     }
 
     function getEnsRegistry() {
-        return profileModelInst.ens.getENSRegistry()
+        return "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e"
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/stores/RootStore.qml:250]: I guess we can put it in Constants at that point

Comment (alaibe) [reply ui/app/AppLayouts/Profile/stores/RootStore.qml:250]: Fair, I've been lazy ;)

**Comment by anastasiyaig** [comment]
@richard-ramos what is the purpose of this PR? I think something is wrong because it shows me that my already registered name is not taken

https://etherscan.io/enslookup-search?search=anastasija.stateofus.eth

Screenshot 2021-12-23 at 15 06 27

**Comment by alaibe** [comment]
@anastasiyaig PR updated

**Comment by anastasiyaig** [comment]
@alaibe there is ENS name data missing when clicking the name

Screenshot 2021-12-24 at 11 14 13

**Review by anastasiyaig — APPROVED** [review]

**Pull request merged**